src/test/solution_space/solution_graph/graph_vertices_dist_tests.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
```

Comment:
Please rename it into `graph_vertices_dist_test`, because for **pytest** files should have prefix or suffix **test**